docs/index.md

```diff
@@ -0,0 +1,18 @@
+title: EthSigner
+description: EthSigner is an open-source Ethereum transaction signer developed under the Apache 2.0 license and written in Java.
+<!--- END of page meta data -->
+
+# EthSigner
+
+## What is EthSigner?
+
+//TODO
+
+## What can you do with EthSigner?
+
+EthSigner includes a [command line interface](Reference/EthSigner-CLI.md)
+//TODO
```

Comment:
Suggestion (as per Madeline's text)

What is EthSigner?
Ethsigner is an open-source, client-agnostic Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 
Ethsigner enables key management to be separated from transaction validation with private key signing.

Reply:
@MadelineMurray content work will override this one, so I just let it as is and we'll merge #59 as soon as possible.